site/gatsby-site/src/components/discover/Hits.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { connectHits, connectStateResults } from 'react-instantsearch-dom';
 import Hit from './Hit';
-import { Spinner } from 'flowbite-react';
 import { DisplayModeEnumParam } from './queryParams';
 import { useQueryParam } from 'use-query-params';
+import ListSkeleton from 'elements/Skeletons/List';
```

Comment:
I think for the compact and detail views, it would be better to use a [card skeleton](https://flowbite.com/docs/components/skeleton/#card-placeholder)